lib/src/domain/authorization/authorization.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:firebase_auth/firebase_auth.dart';
+
+abstract class Authorization {
+
+  Stream<User> userStream();
```

Comment:
w takim klasycznym podejściu pole reprezentuje stan, a metoda zachowanie, więc jak `userStream` to fajnie jakby było polem, jak metoda, to `getUserStream` albo `observeUserChanges` :)

Reply:
observeUserChanges mi się bardziej podoba :)